Sources/WebSocketKit/WebSocket.swift

```diff
@@ -164,21 +200,37 @@ public final class WebSocket: Sendable {
         fin: Bool = true,
         promise: EventLoopPromise<Void>? = nil
     ) {
-        let frame = WebSocketFrame(
-            fin: fin,
-            opcode: opcode,
-            maskKey: self.makeMaskKey(),
-            data: data
-        )
-        self.channel.writeAndFlush(frame, promise: promise)
-    }
+        //using compression
+        if pmce != nil && pmce!.enabled {
```

Comment:
Please `if let` unwrap this, instead of force unwrapping

Reply:
done.